### PR TITLE
German translation for release 1.3

### PR DESCRIPTION
**Pull request by twollgam**
### What does this PR do?

German translation for release 1.3

### Closes Issue(s)

none

### Motivation

Complete german translations.

### More

- [x] Updated german translations without xml
- [x] Updated german translations with xml

The readme file only contains the hint to the wiki article for translation.

### CODE REVIEW

**Comment by Baklap4** [comment]
Converted to draft PR since still in progress

**Comment by twollgam** [comment]
@Baklap4 could you please review the readme and the python script.

File: lang/xml_gettext.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def _start(self, *args, **kwargs):
                 f'msgid "{text}"\n'
                 'msgstr ""\n'
             )
-        if elem.tag in ("enum"):
+        if elem.tag in ("enum") and not elem.text is None:
```
Comment (Baklap4) [inline lang/xml_gettext.py:66]: Do we also need to do this on line 47?

Comment (twollgam) [reply lang/xml_gettext.py:66]: Added the check to line 47.

**Review by Baklap4 — APPROVED** [review]

**Pull request merged**